content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,479 @@
+work`, :guilabel:`My future`, and :guilabel:`My feelings`. The :guilabel:`Manager Feedback Template`
+has the following sections: :guilabel:`Feedback`, :guilabel:`Evaluation`, and
+:guilabel:`Improvements`.
+
+Make any desired changes to the feedback templates by making changes directly in each template.
+
+Appraisals
+~~~~~~~~~~
+
+The :guilabel:`Appraisal` section of the settings menu deals with the frequency that appraisals are
+performed, and if other feedback is available to request.
+
+.. image:: appraisals/appraisals-setting.png
+   :align: center
+   :alt: The appraisals sections with the timeline filled in and 360 feedback enabled.
+
+.. _appraisals/appraisal-plan:
+
+Appraisals plans
+****************
+
+Appraisals are pre-configured to be automatically generated six months after an employee is hired,
+then another appraisal six months after that. After those initial two appraisals that first year,
+appraisals are conducted once a year (every twelve months).
+
+To modify the schedule, change the number of months in the blank fields under the
+:guilabel:`Appraisal Plans` section.
+
+.. important::
+   If the :guilabel:`Appraisals Plans` section is modified, all empty :guilabel:`Next Appraisal
+   Dates` are modified for all employees.
+
+360 feedback
+************
+
+Enable the :guilabel:`360 Feedback` option to allow managers to request feedback from other
+employees using a different survey form, at any time, independent of the appraisal schedule.
+Typically, managers ask for feedback from other people who work with an employee they manage. This
+includes the employee's various manager, peers, and direct reports.
+
+To view the :guilabel:`360 Degree Feedback` survey, click the :guilabel:`→ Internal link` icon after
+the :guilabel:`Default Template` field. The :guilabel:`360 Feedback` survey loads. Make any desired
+changes to the survey here. For more information on how to edit a survey, refer to the
+:doc:`../marketing/surveys/create` document.
+
+.. note::
+   The 360 Feedback form is a pre-configured survey within the *Surveys* application. In order to
+   use the :guilabel:`360 Feedback` option, including the ability to edit the survey, the *Surveys*
+   application must be installed.
+
+Evaluation scale
+----------------
+
+To view the default final rating options that appear on the employee appraisal form, navigate to
+:menuselection:`Appraisals application --> Configuration --> Evaluation Scale`. This presents the
+ratings in a list view. The pre-configured ratings are :guilabel:`Needs Improvement`,
+:guilabel:`Meets Expectations`, :guilabel:`Exceeds Expectations`, and :guilabel:`Strongly Exceeds
+Expectations`. To add another rating, click the :guilabel:`New` button and a blank line appears at
+the bottom of the list. Enter the name of the rating in the field. To rearrange the order of the
+ratings, click on the six small gray boxes to the left of a rating, and drag the rating to the
```

Comment:
It doesn't but the column name is name so I added that as the Gui name for the field. As for the small grey boxes, I don't believe they do- so I described them since I saw this description in another doc as well.